image.c

```diff
@@ -320,14 +372,9 @@ bool img_load_webp(img_t* img, const fileinfo_t* file)
 	uint32_t flags;
 	uint32_t delay;
 
-	/* Check if file exists (and we are able to read it) */
-	if (access(file->path, R_OK) == -1) {
-		error(0, 0, "%s: Error opening webp image", file->name);
-		return false;
-	}
 	/* Open the file */
 	if ((webp_file = fopen(file->path, "rb")) == NULL) {
-		error(0, 0, "%s: Error opening webp image", file->name);
+		error(0, 0, "%s: Error opening webp image (1)", file->name);
```

Comment:
I just realised I forgot to revert these debugging changes. Before the PR is merged I'm going to change the generic error messages to be a bit more specific, so that if there's a problem with the WebP library version the error message will mention that, instead of just saying "error opening webp image".